Doc/library/_thread.rst

```diff
@@ -118,7 +118,7 @@ This module defines the following constants and functions:
    Its value may be used to uniquely identify this particular thread system-wide
    (until the thread terminates, after which the value may be recycled by the OS).
 
-   .. availability:: Windows, FreeBSD, Linux, macOS, OpenBSD, NetBSD, AIX.
+   .. availability:: Windows, FreeBSD, Linux, macOS, OpenBSD, NetBSD, AIX, DragonFlyBSD.
```

Comment:
Nit: is it DragonFly BSD or DragonFlyBSD? From their website, I suspect the former. OTOH, there is no consistent usage in the docs.

Reply:
it s consistent with other BSD doc wise and does not stick out :-)

Reply:
I'm fine with that. If someone wants to normalise DragonFly BSD naming across the docs, feel free to take it on 😄